tasks/task135_winowhy_wrong_reason_generation.json

```diff
@@ -0,0 +1,316 @@
+
+{
+    "Contributors": [
+        "Xinran Zhao",
+        "Hongming Zhang",
+        "Yangqiu Song"
+    ],
+    "Source": [
+        "winowhy"
+    ],
+    "Categories": [
+        "Incorrect Answer Generation"
+    ],
+    "Definition": "In this task you need to give wrong reasons to justify the pronoun coreference relations. The given sentences are provided by a sentence with a target pronoun and a sentence that justifies which noun phrase the pronoun refers to. Good practices involve (1) writing a sentence that gives no reason but relates to the content (e.g., mention the noun phrases that appear in the original sentence); (2) writing a completely unrelated sentence.",
```

Comment:
I don't understand what is the intended meaning of the following sentence is for this task which is about giving **incorrect** reasons: 
"The given sentences are provided by a sentence with a target pronoun and a sentence that justifies which noun phrase the pronoun refers to."

Reply:
Are we really sure about this? "(2) writing a completely unrelated sentence" 
From all your examples/instances, it seems that they're all somehow relevant to the given input.

Reply:
> I don't understand what is the intended meaning of the following sentence is for this task which is about giving incorrect reasons: "The given sentences are provided by a sentence with a target pronoun and a sentence that justifies which noun phrase the pronoun refers to." 

In this task, there is no sentence that "a sentence that justifies which noun ...".

Reply:
Thank you for the comments! I will resolve the ambiguity very soon.